client/lib/wpcom-undocumented/lib/me.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Module dependencies.
  */
-import Me from 'wpcom/dist/lib/me';
+import Me from 'wpcom/build/lib/me';
```

Comment:
> In this case should we sport the all libraries from the core?

I think yes!

As an alternative, we _could_ add symlinks in the root of wpcom.js pointing into the `build/lib` dir, such that we could do `import Me from 'wpcom/me'`, though I'm pretty sure npm doesn't play nicely with symlinks.